spec/features/login_signup/login_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+# rubocop:disable Metrics/BlockLength
 RSpec.feature 'Logins', type: :feature do
```

Comment:
- [ ] Kindly don't disable your linters because linters prevents you from making those error in your code which can break your application completely :fearful: Kindly remove this line and fix your linters manually :+1: